cdc/puller/puller.go

```diff
@@ -205,6 +207,26 @@ func (p *pullerImpl) Run(ctx context.Context) error {
 							zap.Any("spans", p.spans),
 						)
 					}
+					if lastSlowestRange != nil {
```

Comment:
```suggestion
					if lastSlowestRange != nil && time.Since(lastCheckSlowestRangeTime) > 30*time.Second {
```

Reply:
If we check every 30 seconds, there is a high probability that we will not capture the target region. Therefore, we check all regions, and print the message every 30 seconds.